variables.tf

```diff
@@ -140,12 +144,24 @@ variable "highly_available" {
   default     = false
 }
 
+variable "highly_available_read_replica" {
+  description = "Whether the MySQL instance should be highly available REGIONAL or ZONAL. Highly Available (HA) instances will automatically failover to another zone within the region if there is an outage of the primary zone. HA instances are recommended for production use-cases and increase cost"
+  type        = bool
+  default     = false
+}
+
 variable "read_replica_count" {
   description = "Specify the number of read replicas for the MySQL instance. Value greater than 0 requires 'var.pit_recovery_enabled' to be 'true'."
   type        = number
   default     = 0
 }
 
+variable "read_replica_pvt_ip_range" {
+  description = "The name of the allocated ip range for the private ip CloudSQL instance. For example: google-managed-services-default. If set, the instance ip will be created in the allocated range. The range name must comply with RFC 1035. Specifically, the name must be 1-63 characters long and match the regular expression a-z?"
```

Comment:
May use similar description as `allocated_ip_range` variable. Also update the variable name as mention in above comment.

Reply:
Also, the description needs to be revised to be clear that this is for the READ-REPLICA instances and not for the master instances.